test/e2e/basic/available.go

```diff
@@ -0,0 +1,50 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	configv1 "github.com/openshift/api/config/v1"
+	tunedv1 "github.com/openshift/cluster-node-tuning-operator/pkg/apis/tuned/v1"
+	ntoconfig "github.com/openshift/cluster-node-tuning-operator/pkg/config"
+)
+
+var _ = Describe("Node Tuning Operator: availability", func() {
```

Comment:
Suggestion:  change all describe sections according to folder and name , e.g.:
Describe([basic][available]  Node tuning operator availability)

Reply:
Will do, thank you!